nextbike/prediction/predict.py

```diff
@@ -1,28 +1,38 @@
 from .. import io
 from sklearn import metrics
+from tensorflow.keras.models import load_model
 import numpy as np
 
 
 def predict_by_regression(pX_test, py_test):
     scaler = io.read_object("Standard_Scaler.pkl")
     model = io.read_object("Linear_Regression_Model.pkl")
-    pca = io.read_object("PCA.pkl")
+    #pca = io.read_object("PCA.pkl")
```

Comment:
Why dont we use PCA anymore?

Reply:
PCA just captures x % of the variance in the data. Less variance could mean less information from the data. In our case the algorithms perform worse with less variance captured by the principle components. 
We should check at how many PCs we do not lose prediction power.